### PR TITLE
Added days uptime to the status message

### PR DESCRIPTION
**Pull request by NanoAkron**

### CODE REVIEW

**Comment by NanoAkron** [comment]
Oops, hours counter will keep incrementing beyond 23hrs.

**Comment by vtnerd** [comment]
And days only needs to be `uptime / 86400`. std::chrono::steady_clock makes this slightly easier to read if interested, but there is no `std::chrono::days` sadly.

**Comment by NanoAkron** [comment]
I've not got much experience with `<chrono>` I'm afraid, but I read that it's more type-safe and accurate than `<ctime>` which we're currently using in `rpc_command_executor.cpp`.

It seems like switching over would be a good idea, but something for a more experienced programmer (in case I bork the entire hard fork mechanism for example...) ;)

File: src/daemon/rpc_command_executor.cpp [hunk 1]
```diff
@@ -378,9 +378,11 @@ bool t_rpc_command_executor::show_status() {
...
     % get_fork_extra_info(hfres.earliest_height, ires.height, ires.target)
     % (hfres.state == cryptonote::HardFork::Ready ? "up to date" : hfres.state == cryptonote::HardFork::UpdateNeeded ? "update needed" : "out of date, likely forked")
-    % (unsigned)ires.outgoing_connections_count % (unsigned)ires.incoming_connections_count
-    % (unsigned int)floor(uptime / 3600.0)
-    % (unsigned int)floor(fmod(uptime, 3600.0) / 60.0)
+    % (unsigned)ires.outgoing_connections_count
+    % (unsigned)ires.incoming_connections_count
+    % (unsigned int)floor(uptime / 60.0 / 60.0 / 24.0)
+    % (unsigned int)floor(fmod((uptime / 60.0 / 60.0), 24.0)
+    % (unsigned int)floor(fmod((uptime / 60.0), 60.0)
```
Comment (tdprime) [inline src/daemon/rpc_command_executor.cpp]: Unbalanced braces on the last 2 lines.  4 left, 3 right.  Cause error caught by compiler.
/home/tdprime/bitmonero/src/daemon/rpc_command_executor.cpp: In member function ‘bool daemonize::t_rpc_command_executor::show_status()’:
/home/tdprime/bitmonero/src/daemon/rpc_command_executor.cpp:386:38: error: invalid operands of types ‘double’ and ‘unsigned int’ to binary ‘operator%’
     % (unsigned int)fmod(uptime, 60.0)
     ^

**Pull request merged**